Library/Homebrew/cmd/update

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh # brew-update.sh
+[ -n "$HOMEBREW_VERBOSE" ] && set -x
+
+trap_cleanup()
+{
+  for job in $(jobs -p); do
```

Comment:
I'm not super familiar with jobs so am probably wrong but: could this not affect other jobs the user might have running in their environment?